Classes/Controller/BackendServiceController.php

```diff
@@ -545,4 +552,18 @@ function ($envelope) {
 
         return json_encode($result);
     }
+
+    /**
+     * Generates a new uri path segment for the given node and title
+     *
+     * @param NodeInterface $contextNode
+     * @param string $text
+     * @return string
+     * @throws \Neos\Neos\Exception
+     */
+    public function transliteratePathSegmentAction(NodeInterface $contextNode, string $text): string
```

Comment:
Should be void as we use the JSON view, you can remove the param and returns parts in the doc comment.

```suggestion
    public function transliteratePathSegmentAction(NodeInterface $contextNode, string $text): void
```